Source/Core/Scripting/Python/Modules/utilmodule.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include <Python.h>
+
+namespace PyScripting
+{
+    PyMODINIT_FUNC PyInit_dol_utils();
+}
```

Comment:
Newline.

Reply:
I believe I resolved this, but I'm not sure exactly what you meant lol